spec/controllers/saml_idp_controller_spec.rb

```diff
@@ -44,6 +44,70 @@
 
       delete :logout, params: { SAMLRequest: 'foo' }
     end
+
+    let(:service_provider) do
+      create(:service_provider,
+             cert: nil, # override singular cert
+             certs: ['saml_test_sp'],
+             active: true,
+             assertion_consumer_logout_service_url: 'https://example.com')
+    end
+
+    let(:right_cert_settings) do
+      sp1_saml_settings.tap do |settings|
```

Comment:
**Question:** Is it worth adding a comment here to specify that the `saml_test_sp` cert is already selected in the setup of `sp1_saml_settings`? Feels a little like a mystery guest that could be annoying to figure out in the future.

Reply:
My thinking was that literally all of the certs in the test are that first cert, so I think it's fine?

Reply:
Yeah that's totally fine - I tend to be a little generous with comments personally but that's just me 😄